svelte-kit-scss/src/lib/components/Profile/common/SocialLink.svelte

```diff
@@ -5,6 +5,7 @@
   export let label: string | null | undefined = undefined;
   export let iconComponent: ComponentType;
   export let mobileHide = false;
+  export let testId: string | null | undefined = undefined;
```

Comment:
`export let testId?: string | null` instead? Noticed that both the current syntax and this are used across the codebase, is it possible to pick a single approach? The suggestion here feels a bit more succinct